docs/test_instructions_per_dependency_update.md

```diff
@@ -0,0 +1,870 @@
+
+ℹ️ Just test the most recent experimental feature.
+
+</details>
+
+-----
+
+### Lottie [[lottieVersion](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/build.gradle)] <a name="lottie"></a>
+
+<details>
+    <summary>1. [JP/WP] JP Install Full Plugin Animation [JPInstallFullPluginAnimation.kt]</summary>
+
+- TODO
+- TODO
+- TODO
+
+</details>
+
+<details>
+    <summary>1. [WP] Jetpack Static Poster [JetpackStaticPoster.kt]</summary>
+
+- TODO
+- TODO
+- TODO
+
+</details>
+
+<details>
+    <summary>1. [JP] Notifications Screen [NotificationsDetailListFragment.kt]</summary>
+
+- TODO
+- TODO
+- TODO
+
+</details>
+
+-----
+
+### UCrop [[uCropVersion](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/build.gradle)] <a name="ucrop"></a>
+
+<details>
+    <summary>1. [JP/WP] Image Edit Screen [PreviewImageFragment.kt + CropFragment.kt]</summary>
+
+- Add a new `blog` post.
+- Add a new `image` block.
+- Choose an image and wait for it to be uploaded within the `image` block.
+- Click on the `media options` of this image (top right) and then click `edit`.
+- Crop the image and click the `done` menu option (top right).
+- Verify the image is updated accordingly.
+
+</details>
+
+-----
+
+## Smoke Test <a name="smoke-test"></a>
+
+1. Plugin
+    1. [GoogleServices](#googleservices)
+    2. [Dagger](#dagger)
+2. Kotlin
+    1. [Kotlin](#kotlin)
+    2. [Coroutines](#coroutines)
+3. EventBus
+    1. [EventBus](#eventbus)
+4. AndroidX Core
+    1. [Annotation](#annotation)
+    2. [ArchCore](#archcore)
+    3. [Lifecycle](#lifecycle)
+    4. [Core](#core)
+    5. [Activity](#activity)
+    6. [Fragment](#fragment)
+    7. [AppCompat](#appcompat)
+5. AndroidX Compose
+    1. [Compose](#compose)
+    2. [ComposeMaterial3](#composematerial3)
+    3. [ComposeConstraintLayout](#composeconstraintlayout)
+    4. [Coil](#coil)
+6. AndroidX/Google
+    1. [RecyclerView](#recyclerview)
+    2. [ViewPager2](#viewpager2)
+7. Material/Google
+    1. [Material](#material)
+    2. [ConstraintLayout](#constraintlayout)
+8. Network
+    1. [Retrofit](#retrofit)
+    2. [Volley](#volley)
+    3. [Glide](#glide)
+    4. [Gson](#gson)
+9. Tool
+    1. [InstallReferrer](#installreferrer)
+10. Other Core
+    1. [ApacheCommons](#apachecommons)
+11. Other
+    1. [Desugar](#desugar)
+
+ℹ️ Some smoke test instructions might have an `Extra` section with addition instructions.
+
+-----
+
+### GoogleServices [[googleServicesVersion](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/settings.gradle#1)] <a name="googleservices"></a>
+
+-----
+
+### Dagger [[daggerVersion](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/settings.gradle#1)] <a name="dagger"></a>
+
+-----
+
+### Kotlin [[kotlinVersion](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/settings.gradle#1)] + [[androidxComposeCompilerVersion](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/build.gradle)] <a name="kotlin"></a>
```

Comment:
np: `googleServicesVersion`, `daggerVersion`, and `kotlinVersion` links still contain `#1`. We can remove them and link `https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/settings.gradle` file for consistency.

Reply:
OMG, sorry for that @irfano and thanks for being so thorough here, I replaced all usages of `#L1`, but I also had `#1` there, whot... 🤷

Reply:
Done: 10796939f0c60174d453163661bf6e28fd1b350c